### ISSUE TITLE
Calibnet cannot parse message details using 1.24.0-rc3

### ISSUE THREAD

**Issue by simlecode** [filecoin-project/lotus#11372]
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [ ] I did not make any code changes to lotus.

### Lotus component

- [ ] lotus daemon - chain sync
- [ ] lotus fvm/fevm - Lotus FVM and FEVM interactions
- [ ] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt/WinningPoSt)
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [X] Other

### Lotus Version

```text
1.24.0-rc2+calibnet+git.60f78c792 and 1.24.0-rc3+calibnet+git.6f9e02fd7
```

### Repro Steps

1. Run '...'
2. Do '...'
3. See error '...'
...

### Describe the Bug

Executing `./lotus-shed msg bafy2bzacebffwml7ybb3od7tpd2tl2gc7gdyslindxuaf2jvupaqmckj6zx6m` in `calibnet` has different results in `v1.24.0-rc2` and `v1.24.0-rc3`

```
1.24.0-rc3+calibnet+git.6f9e02fd7

Message Details:
Value: 0 FIL
Max Fees: 0.000002807280505575 FIL
Max Total Cost: 0.000002807280505575 FIL
Method:
Params: raw:85078182004081820d58c0b219288afa30303d47cbc82389d5856205a7a1e3d2fe3f8a70e38c7ea67d7791562065540734fab06443e25ead52a6a6ab9d6510b454a86f69ad1773bc82a7a6cc17399cf6fb88ccf952720b4904661526cf8ee0e55cefb818d54f4e1a4cf2d5166fd2b9b999edcaecd854586e611e04cce0f21128e23a2990e7e2c636fff29aec1a1ee5395e9c6155ba3bb091da2d7a9814e5ce4bad4b60f7286acf313ead2aa73e2d4ab40dd360496044cbebb456a60a11c005f8fe21a730ecd63f8676d77e1a001003175820a4b097a1470a9526c9e329ecb1969a0bb7b5bb943352f90103164326fa72a7d2
```

```
1.24.0-rc2+calibnet+git.60f78c792

Message Details:
Value: 0 FIL
Max Fees: 0.000002807280505575 FIL
Max Total Cost: 0.000002807280505575 FIL
Method: SubmitWindowedPoSt
Params: {
  "Deadline": 7,
  "Partitions": [
    {
      "Index": 0,
      "Skipped": [
        0
      ]
    }
  ],
  "Proofs": [
    {
      "PoStProof": 13,
      "ProofBytes": "shkoivowMD1Hy8gjidWFYgWnoePS/j+KcOOMfqZ9d5FWIGVUBzT6sGRD4l6tUqamq51lELRUqG9prRdzvIKnpswXOZz2+4jM+VJyC0kEZhUmz47g5VzvuBjVT04aTPLVFm/SubmZ7crs2FRYbmEeBMzg8hEo4jopkOfixjb/8prsGh7lOV6cYVW6O7CR2i16mBTlzkutS2D3KGrPMT6tKqc+LUq0DdNgSWBEy+u0VqYKEcAF+P4hpzDs1j+Gdtd+"
    }
  ],
  "ChainCommitEpoch": 1049367,
  "ChainCommitRand": "pLCXoUcKlSbJ4ynssZaaC7e1u5QzUvkBAxZDJvpyp9I="
}
```

### Logging Information

```text
.
```

**Comment by rjan90** [comment]
Can confirm that I also cannot parse the message details with v1.24.0-rc3 on Calibration: 

```
./lotus-shed msg bafy2bzacebffwml7ybb3od7tpd2tl2gc7gdyslindxuaf2jvupaqmckj6zx6m
Unsigned:
CID: bafy2bzacebffwml7ybb3od7tpd2tl2gc7gdyslindxuaf2jvupaqmckj6zx6m
HEX: 8a004300a71d583103aec0f9a8d6945fa9d5e34564eb709406ac444ff79fed8a36f30f399be637b6a5598a8795b94d30fe354e271a15bc6c5619be30401a01a694914400018bf744000187d90558f285078182004081820d58c0b219288afa30303d47cbc82389d5856205a7a1e3d2fe3f8a70e38c7ea67d7791562065540734fab06443e25ead52a6a6ab9d6510b454a86f69ad1773bc82a7a6cc17399cf6fb88ccf952720b4904661526cf8ee0e55cefb818d54f4e1a4cf2d5166fd2b9b999edcaecd854586e611e04cce0f21128e23a2990e7e2c636fff29aec1a1ee5395e9c6155ba3bb091da2d7a9814e5ce4bad4b60f7286acf313ead2aa73e2d4ab40dd360496044cbebb456a60a11c005f8fe21a730ecd63f8676d77e1a001003175820a4b097a1470a9526c9e329ecb1969a0bb7b5bb943352f90103164326fa72a7d2
B64: igBDAKcdWDEDrsD5qNaUX6nV40Vk63CUBqxET/ef7Yo28w85m+Y3tqVZioeVuU0w/jVOJxoVvGxWGb4wQBoBppSRRAABi/dEAAGH2QVY8oUHgYIAQIGCDVjAshkoivowMD1Hy8gjidWFYgWnoePS/j+KcOOMfqZ9d5FWIGVUBzT6sGRD4l6tUqamq51lELRUqG9prRdzvIKnpswXOZz2+4jM+VJyC0kEZhUmz47g5VzvuBjVT04aTPLVFm/SubmZ7crs2FRYbmEeBMzg8hEo4jopkOfixjb/8prsGh7lOV6cYVW6O7CR2i16mBTlzkutS2D3KGrPMT6tKqc+LUq0DdNgSWBEy+u0VqYKEcAF+P4hpzDs1j+Gdtd+GgAQAxdYIKSwl6FHCpUmyeMp7LGWmgu3tbuUM1L5AQMWQyb6cqfS
JSON: {
  "Version": 0,
  "To": "t03751",
  "From": "t3v3aptkgwsrp2tvpdivsow4eua2weit7xt7wyunxtb44zxzrxw2svtcuhsw4u2mh6gvhcogqvxrwfndfajeka",
  "Nonce": 48688,
  "Value": "0",
  "GasLimit": 27694225,
  "GasFeeCap": "101367",
  "GasPremium": "100313",
  "Method": 5,
  "Params": "hQeBggBAgYINWMCyGSiK+jAwPUfLyCOJ1YViBaeh49L+P4pw44x+pn13kVYgZVQHNPqwZEPiXq1SpqarnWUQtFSob2mtF3O8gqemzBc5nPb7iMz5UnILSQRmFSbPjuDlXO+4GNVPThpM8tUWb9K5uZntyuzYVFhuYR4EzODyESjiOimQ5+LGNv/ymuwaHuU5XpxhVbo7sJHaLXqYFOXOS61LYPcoas8xPq0qpz4tSrQN02BJYETL67RWpgoRwAX4/iGnMOzWP4Z2134aABADF1ggpLCXoUcKlSbJ4ynssZaaC7e1u5QzUvkBAxZDJvpyp9I=",
  "CID": {
    "/": "bafy2bzacebffwml7ybb3od7tpd2tl2gc7gdyslindxuaf2jvupaqmckj6zx6m"
  }
}

---
Message Details:
Value: 0 FIL
Max Fees: 0.000002807280505575 FIL
Max Total Cost: 0.000002807280505575 FIL
Method: 
Params: raw:85078182004081820d58c0b219288afa30303d47cbc82389d5856205a7a1e3d2fe3f8a70e38c7ea67d7791562065540734fab06443e25ead52a6a6ab9d6510b454a86f69ad1773bc82a7a6cc17399cf6fb88ccf952720b4904661526cf8ee0e55cefb818d54f4e1a4cf2d5166fd2b9b999edcaecd854586e611e04cce0f21128e23a2990e7e2c636fff29aec1a1ee5395e9c6155ba3bb091da2d7a9814e5ce4bad4b60f7286acf313ead2aa73e2d4ab40dd360496044cbebb456a60a11c005f8fe21a730ecd63f8676d77e1a001003175820a4b097a1470a9526c9e329ecb1969a0bb7b5bb943352f90103164326fa72a7d2
```
--------
However, I do not encounter the same issue on the Mainnet with Lotus v1.24.0-rc3:
```
./lotus-shed msg bafy2bzacecbcxnvs3ke72iu7widehd4lucekfijmq2mdjrokqig5q5xv2gil2
---
Message Details:
Value: 0 FIL
Max Fees: 0.025823204943502164 FIL
Max Total Cost: 0.025823204943502164 FIL
Method: SubmitWindowedPoSt
Params: {
  "Deadline": 14,
  "Partitions": [
    {
      "Index": 0,
      "Skipped": [
        0
      ]
    }
  ],
  "Proofs": [
    {
      "PoStProof": 13,
      "ProofBytes": "hiAyCdilh0A72naQFVvH7dlqlEaUrhtLHlSw2UGWwE3uCjfv19TCMK4+tpVn9k/XiGax+YKWtSSjJli3XEzero0QZAotX+L6RbaMnqDqbVyEKwx1MTMCiYN/aPMTYL97BSckLRVnjUGqw3KI0KlWumUMT5v4CTo6ZWZW6PYA0PvXuZDltMW8wfjWHDUetl3op9oP712sfYraqxM3l7V4o6HQ3lwt4+gTxDIMWYvXax7aM1y70Z4UM4FC9p4y69S+"
    }
  ],
  "ChainCommitEpoch": 3350672,
  "ChainCommitRand": "rykEm7iUq+TsdtSYfX2rcpxJ+6IyK6FpsKX/O1UdWDk="
}
```

**Comment by arajasek** [comment]
Thanks for the report! I've confirmed the issue exists as reported.

This _should_ only be an issue for miner-related messages on calibnet between the heights 1013134 and 1070494 (about 20 days). Is it okay to leave unfixed, given that it won't affect mainnet / calibnet after 1070494?

We certainly _can_ fix it, the only issue is that doing so requires special-casing the "wrong" miner CID in more places than strictly necessary (we already did some hard-coding of that in #11363, and I'd prefer not to do more).
